packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -127,10 +128,22 @@ const CFDsListing = observer(() => {
         }
     };
 
-    const hasUnavailableAccount = combined_cfd_mt5_accounts.some(account => account.status === 'unavailable');
-    const hasMaintenanceStatus = combined_cfd_mt5_accounts.some(account => account.status === 'under_maintenance');
+    const hasUnavailableAccount = combined_cfd_mt5_accounts.some(
+        account => account.status === TRADING_PLATFORM_STATUS.UNAVAILABLE
+    );
+    const hasMaintenanceStatus =
+        combined_cfd_mt5_accounts.some(account => account.status === MT5_ACCOUNT_STATUS.UNDER_MAINTENANCE) ||
+        combined_cfd_mt5_accounts.some(
+            account => getPlatformStatus(account.platform ?? CFD_PLATFORMS.MT5) === TRADING_PLATFORM_STATUS.MAINTENANCE
+        );
 
     const getMT5AccountAuthStatus = (current_acc_status?: string | null, jurisdiction?: string) => {
+        if (current_acc_status === 'under_maintenance') {
+            return MT5_ACCOUNT_STATUS.UNDER_MAINTENANCE;
```

Comment:
Can't we use the same constant for these statuses ? Getting one from MT5_ACCOUNT_STATUS and the other one from TRADING_PLATFORM_STATUS does not make sense since both of them are giving us the server status.

Reply:
Thanks for explanation @shontzu-deriv . We can talk with BE team to give us `under_maintenance` for wherever it is passed instead of `maintenance` in one call and another name in other calls.